test/AWS.Deploy.CLI.IntegrationTests/WebAppNoDockerFileTests.cs

```diff
@@ -75,9 +75,9 @@ public async Task DefaultConfigurations()
             Assert.False(Directory.Exists(tempCdkProject), $"{tempCdkProject} must not exist.");
 
             // Example:     Endpoint: http://52.36.216.238/
-            var applicationUrl = deployStdOut.First(line => line.Trim().StartsWith($"Endpoint"))
```

Comment:
Noticed the logic for testing the HTTP endpoint was invalid because when the line is split on `:` there are actually 3 tokens. The logic was forgetting there as a `:` in the http://endpoint.com/ part.